plash_cli/core.py

```diff
@@ -55,7 +55,7 @@ def is_included(path):
     return not any(p in excludes for p in path.parts)
 
 # %% ../nbs/00_core.ipynb 10
-def create_tar_archive(path # Path to directory containing FastHTML app
+def _tarz(path # Path to directory containing FastHTML app
```

Comment:
Is function `_tarz` defined twice (L 58, 124) on purpose? I dont think I get why.